src/config/app.spec.ts

```diff
@@ -0,0 +1,32 @@
+import {PathLike} from 'fs';
+import createConfig from './app';
+
+describe('config/app', () => {
+    describe('createConfig()', () => {
+        const phpIniFromConfigurationPath: PathLike = 'tests/php-ini-from-configuration';
+
+        it('should return valid config', () => {
+            expect(createConfig(
```

Comment:
Overall okay with this testing approach, but note that we could've caught this with the previous testing approach too 👍

Reply:
I think of the current approach as higher up the pyramid. Unit testing will give us quicker clarity as to what the issue is, so the more the better I say.